simde/x86/avx512/popcnt.h

```diff
@@ -0,0 +1,49 @@
+#if !defined(SIMDE_X86_AVX512_POPCNT_H)
+#define SIMDE_X86_AVX512_POPCNT_H
+
+#include "types.h"
+
+HEDLEY_DIAGNOSTIC_PUSH
+SIMDE_DISABLE_UNWANTED_DIAGNOSTICS
+SIMDE_BEGIN_DECLS_
+
+SIMDE_FUNCTION_ATTRIBUTES
+simde__m128i
+simde_mm_popcnt_epi16 (simde__m128i a) {
+  #if defined(SIMDE_X86_AVX512BITALG_NATIVE) && defined(SIMDE_X86_AVX512VL_NATIVE)
+    return _mm_popcnt_epi16(a);
+  #else
+    simde__m128i_private
+      r_,
+      a_ = simde__m128i_to_private(a);
+
+    #if defined(SIMDE_VECTOR_SUBSCRIPT_SCALAR)
+      a_.u16 -= ((a_.u16 >> 1) & 21845);
+      a_.u16 = ((a_.u16 & 13107) + ((a_.u16 >> 2) & 13107));
```

Comment:
I wouldn't even bother going back and adding a patch for this now, but in the future, similarly to your other PR where I suggested lining up the arguments, lining up the assignments is probably a good idea here:

```c
a_.u16 -= ...
a_.u16  = ...
```

Unfortunately the operations don't really line up, not much you can do there.